### DISCUSSION TITLE
Make the Display Conditions modal wider (Can't see document names)

### DISCUSSION THREAD

**Discussion by shayeryan** [elementor/elementor#22143] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When adding a condition to a custom code for example. The page title gets cut off on the right hand side and you can tell which pages you have the condition set on. This problem becomes prevalent when you drill down more to a singular page and/or have parent pages.

**BEFORE**
![image](https://user-images.githubusercontent.com/7247532/233496443-e75b5dc7-0bdb-4d35-99be-f40a21cb99dc.png)

### Describe the solution you'd like

Make it wider so you can read what page it is. This is after modifying CSS. 

**AFTER**
![image](https://user-images.githubusercontent.com/7247532/233495766-398b179a-50fa-4197-ad40-8c6f662b3c77.png)

### Describe alternatives you've considered

Modifying CSS with Dev tools

**Comment by black-eye** [comment]
Oh lord, yes! I created the very same feature request more than 2 years ago: https://github.com/elementor/elementor/issues/13525
It would be so easy to fix...!

**Reply by shayeryan** [reply]
Oh cool, I didn't see your request before posting. Maybe this will get it on the radar again!

**Comment by fabigrafo** [comment]
Or add a hover text like in the new topbar. It's only annoying on complex website, but would be nice.

**Reply by shayeryan** [reply]
Yes for sure!

**Comment by nicholaszein** [comment]
This request was referenced in the following issues and Discussions:

- #13525
